python/tvm/script/tir/__init__.pyi

```diff
@@ -128,6 +128,18 @@ def store(
     var: Var, index: PrimExpr, value: PrimExpr, predicate: Union[PrimExpr, builtins.bool] = True
 ) -> None: ...
 def comm_reducer(lambda_io: Callable[[Any, Any], Any], identities: List[PrimExpr]) -> PrimExpr: ...
+def preflattened_buffer(
+    buf: Var,
```

Comment:
The first argument should be a `Buffer`, so that it can be matched to the same `Buffer` object as appears in the `buffer_map`.

Reply:
Done!

Reply:
LGTM!